pkg/action/executor/k8s_controller.go

```diff
@@ -154,7 +152,19 @@ func (c *parentController) update(updatedSpec *k8sControllerSpec) error {
 		if c.ormClient == nil {
 			return fmt.Errorf("failed to execute action with nil ORMClient")
 		}
-		err = c.ormClient.Update(origControllerObj, c.obj, ownerInfo)
+		resourcePaths, err := getResourcePath(kind, updatedSpec)
+		if err != nil {
+			return fmt.Errorf("failed to execute action, Unable to get resource paths: %v", err)
```

Comment:
Nit: I think we can be concise on the error message. We have error logging on the top level caller already:

```go
fmt.Errorf("uable to get resource paths: %v", err)
```

Reply:
Don't we want to know that this error is related to failure while executing the action? that's the purpose of adding 'failed to execute action' string

Reply:
If you locate the top level caller, it seems to me that the error message is detailed enough:

```go
	glog.V(2).Infof("Begin to resize workload controller %s/%s.", controllerUpdater.namespace, controllerUpdater.name)
	err = controllerUpdater.updateWithRetry(&controllerSpec{0, specs})
	if err != nil {
		glog.Errorf("Failed to resize workload controller %s/%s.", controllerUpdater.namespace, controllerUpdater.name)
		return err
	}
```